src/developer/web-api/tracker.md

```diff
@@ -2976,8 +2976,10 @@ with an access level of *PROTECTED*. Any user with the org unit owner within the
 temporarily access the program-related data by providing a reason for accessing it.
 
 This act of temporarily gaining access is termed *breaking the glass*.
-Currently, temporary access is granted for 3 hours. DHIS2 [audits](https://docs.dhis2.org/en/manage/concepts/audit.html) breaking the glass along with the
-reason specified by the user. This information is also stored in the database, but only if the tracked entity type is configured to allow auditing, which is disabled by default.
+Currently, temporary access is granted for 3 hours. DHIS2 
+[audit](../../sysadmin/concepts/audit.md) breaking the glass along with the
```

Comment:
Have you tested that this works with our docs-builder? 🤔

Reply:
I tried with VS and it works.